### DISCUSSION TITLE
Transfering an active stake/delegation to another account

### DISCUSSION THREAD

**Discussion by raulillo82** [crypto-org-chain/chain-main#448] [Ideas] [answered]
I was wondering whether it'd be possible to introduce a feature to transfer an active stake (delegation) from CRO account A to CRO account B. Account B would gain control of it and they could initiate any possible action such as redelegations, unbonding, etc.
I can think of use cases for both the possibilities that account B has the same owner (for validators owners, it would fix the selfstake amount when delegating from different accounts by mistake, without being affected by the unstake period), or a different owner (that would be like transfering a savings account in the FIAT world).

Should this make sense, let me know in order to open an issue with the feature request (and also let me know if it does belong in this chain-main project or in other). Thanks in advance for checking this!

**Comment by tomtau** [comment, marked as answer]
Thanks for your idea! I'm not sure if I'm clear on the use cases of this change (a sort of "transitive delegation"?). My hunch, however, is that this may be a bit complex due to the way how things are set up internally in Cosmos SDK (how rewards, unbonding, redelegations etc. are tracked) -- you can try opening an issue there: https://github.com/cosmos/cosmos-sdk/issues
There are a few related efforts that you may want to check out:

1. CW proxy contract: see https://medium.com/confio/introducing-the-mask-41d11e51bccf (and https://github.com/crypto-org-chain/chain-main/blob/master/doc/architecture/adr-001.md ) -- this can e.g. address the use case of validator operator key rotations.
2. liquid staking (I guess that can be somewhat realised with the proxy contract): https://github.com/ChorusOne/liquid-staking/blob/master/awesome-liquid-staking.md
3. "epoched staking": https://github.com/cosmos/cosmos-sdk/discussions/8328 this is a mechanism different from the current SDK's staking module... and it may potentially be easier to implement it in there.
